src/python/pants/backend/jvm/tasks/bootstrap_jvm_tools.py

```diff
@@ -237,17 +237,26 @@ def _check_underspecified_tools(self, jvm_tool, targets):
         default option was provided to use for bootstrap.
         """.format(jvm_tool.key)))
 
+  @property
```

Comment:
Why not an `@abstractproperty`? (And `execute_resolve` an `@abstractmethod`?

Reply:
Because this is already an `@abstractproperty` by virtue of the `IvyTaskMixin` and `CoursierMixin` subclassing `JvmResolverBase` -- if these methods aren't explicitly overridden with a non-abstract method/property, you get:
```
Exception message: Can't instantiate abstract class BootstrapJvmTools_bootstrap_bootstrap_jvm_tools with abstract methods resolver_name
```